### DISCUSSION TITLE
GPS coordinates of example scenes

### DISCUSSION THREAD

**Discussion by taha-yassine** [NVlabs/sionna#220] [Q&A]
Hi,

Is it possible to obtain the GPS coordinates of  the example scenes (etoile, munich)?

Thanks!

**Comment by bchateli** [comment]
I am interested as well ! It would be useful to have the GPS coordinates of each corner of the example scenes for plot purposes.

Thanks!

**Comment by faycalaa** [comment]
Hi,

Unfortunately, we didn’t log the coordinates of the scene corners. But, all the geometries came from OpenStreetMap (OSM), and using the [Blender OSM extension](https://prochitecture.gumroad.com/l/blender-osm), you could [recreate a similar scene](https://youtu.be/7xHLDxUaQ7c?si=MvRtSQNgZpbS3lx7) and keep track of the required coordinates:

- The [etoile scene](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.scene.etoile) contains the area around the [Arc de Triomphe in Paris: (48.8737, 2.29507) on OpenStreetMap](https://www.openstreetmap.org/?mlat=48.87372&mlon=2.29507#map=17/48.87372/2.29507).

- The [munich scene](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.scene.munich) contains the area around the [Frauenkirche in Munich: (48.13866, 11.57361) on OpenStreetMap](https://www.openstreetmap.org/?mlat=48.13857&mlon=11.57359#map=18/48.13857/11.57359).
